### DISCUSSION TITLE
Eink display

### DISCUSSION THREAD

**Discussion by reinzler** [jgauchia/IceNav-v3#264] [General]
Hi, thanks for awesome project! I want to know, is it possible to use eink display? If yes, with which resolution? Thanks in advance!

**Comment by jgauchia** [comment]
Hi, first of all, thank you!
It would be possible to use eInk, although I currently don’t have one. The only drawback would be the refresh rate of some screens (like the compass, map rotation, etc.), but it could be adapted. As for resolutions, the optimal one is 320x480, although it’s true that smaller or larger resolutions are also possible.

My intention is to gradually support other existing boards on the market and keep implementing new features in the future. The time I dedicate to this personal project is somewhat limited, so any help is welcome, hahaha.

If you want, you can create an Issue with the requirements, features, etc., and I’ll take it into account.
